tests/helpers/__init__.py

```diff
@@ -1,19 +1,30 @@
+from typing import Dict, TypedDict
+
+from typing_extensions import NotRequired
+
+
+class _FixtureSource(TypedDict):
+    url: str
+    fixture_path: str
+    commit_hash: NotRequired[str]
+    strip_components: NotRequired[int]
+
+
 # Update the links and commit has in order to consume
 # newer/other tests
-TEST_FIXTURES = {
+TEST_FIXTURES: Dict[str, _FixtureSource] = {
     "evm_tools_testdata": {
         "url": "https://github.com/gurukamath/evm-tools-testdata.git",
         "commit_hash": "792422d",
         "fixture_path": "tests/fixtures/evm_tools_testdata",
     },
     "ethereum_tests": {
         "url": "https://github.com/ethereum/tests.git",
-        "commit_hash": "afed83b",
+        "commit_hash": "07f85aafbf2966b4db2b7d9cdf55248f91661ef9",
```

Comment:
`tox -e py3` ends up throwing `stderr: 'fatal: reference is not a tree: 07f85aafbf2966b4db2b7d9cdf55248f91661ef9'` on my local